app/models/solr_document.rb

```diff
@@ -30,12 +50,13 @@ def extents_information
     end
   end
 
-  # self.unique_key = 'id'
-
-  # DublinCore uses the semantic field mappings below to assemble an OAI-compliant Dublin Core document
-  # Semantic mappings of solr stored fields. Fields may be multi or
-  # single valued. See Blacklight::Document::SemanticFields#field_semantics
-  # and Blacklight::Document::SemanticFields#to_semantic_values
-  # Recommendation: Use field names from Dublin Core
-  use_extension(Blacklight::Document::DublinCore)
+  def extract_notes_by_header(header)
+    # compares against the parameterized value of the header to ignore case and punctuation
+    # rubocop:disable Rails/OutputSafety
+    notes.select { |note| JSON.parse(note)["head"].parameterize == I18n.t("ead_notes.#{header}").parameterize }
+      .map { |note| JSON.parse(note)["p"] }
+      .flatten
+      .map { |para| "<p>#{para}</p>".html_safe }
```

Comment:
Wrapping this in `<p/>` is ugly, I'm looking for a nicer way to do this, but this will do for now.